### PR TITLE
Fix Music loading bug for friendz

### PR DESCRIPTION
**Pull request by BlackMinou**
Added "music.ogg" to the res folder, android is aparently not able to load a .ogg from the assets.
I will investigate on the problem for a next PR

### CODE REVIEW

**Comment by privat** [comment]
+1

File: lib/android/audio.nit [hunk 1]
```diff
@@ -537,8 +536,8 @@ redef class Sound
 	end
 
 	redef fun play do
-		if self.error != null then return
 		if not is_loaded then load
+		if self.error != null then return
```
Comment (xymus) [inline lib/android/audio.nit]: Why moving these 2 lines? it will try to load the file at each usage even if it doesn't exist...

**Comment by R4PaSs** [comment]
:shipit:

File: lib/android/audio.nit (hunk 1 above)
Comment (BlackMinou) [inline lib/android/audio.nit]: Solved face to face

**Pull request merged**